packages/shared/lib/clients/orchestrator.ts

```diff
@@ -597,25 +611,7 @@ export class Orchestrator {
             }
 
             if (res.isErr()) {
-                await createActivityLogMessageAndEnd({
-                    level: 'error',
-                    environment_id: connection.environment_id,
-                    activity_log_id: activityLogId,
-                    timestamp: Date.now(),
-                    content: `Failed with error ${res.error.type} ${JSON.stringify(res.error.payload)}`
-                });
-                await logCtx.error(`Failed with error ${res.error.type}`, { payload: res.error.payload });
-                await createActivityLogMessageAndEnd({
-                    level: 'error',
-                    environment_id: connection.environment_id,
-                    activity_log_id: activityLogId,
-                    timestamp: Date.now(),
-                    content: `The post connection script workflow ${workflowId} did not complete successfully`
-                });
-                await logCtx.error(`The post connection script workflow ${workflowId} did not complete successfully`);
-                await logCtx.failed();
-
-                return res;
+                throw res.error;
```

Comment:
forgot this one in my fix this morning https://github.com/NangoHQ/nango/pull/2278